### ISSUE TITLE
[CRASH] Olive gets stuck on export and Crashes

### ISSUE THREAD

**Issue by RachitKeertiDas** [olive-editor/olive#1472]
**Commit Hash** 
19eabf28 - App Image

**Platform** 
Ubuntu 20.10

**Summary**
Olive gets stuck at a certain percentage while exporting (67% for me). Closing the export dialog/ cancelling the export crashes Olive.

**Steps to Reproduce**

1. Open Olive
2. Import a Project
3. Try to Export
4. Export gets stuck, close the Export Dialog

<details><summary><strong>Backtrace</strong></summary><pre><code>

```
#0  0x00000000005b88f8 in olive::rational::operator*= (this=this@entry=0x7fffd1ffa720, rhs=...) at ../app/common/rational.cpp:195
#1  0x00000000005b89c4 in olive::rational::operator* (this=this@entry=0x7fffd1ffa740, rhs=...) at ../app/common/rational.cpp:228
#2  0x00000000005b9241 in olive::Timecode::timestamp_to_time (timestamp=@0x3b600c0: 1995, timebase=...) at ../app/common/rational.h:39
#3  0x000000000069d103 in olive::ExportTask::FrameDownloaded (this=this@entry=0x3b5fe90, f=std::shared_ptr<olive::Frame> (use count 2, weak count 0) = {...}, hash=..., times=..., 
    job_time=job_time@entry=1611333522731) at ../app/render/videoparams.h:108
#4  0x00000000006a845f in olive::RenderTask::Render (this=this@entry=0x3b5fe90, manager=<optimized out>, video_range=..., audio_range=..., mode=mode@entry=olive::RenderMode::kOnline, 
    cache=cache@entry=0x0, force_size=..., force_matrix=..., force_format=<optimized out>, force_color_output=...) at /opt/rh/devtoolset-9/root/usr/include/c++/9/bits/shared_ptr_base.h:1169
#5  0x000000000069c542 in olive::ExportTask::Run (this=0x3b5fe90) at /opt/rh/devtoolset-9/root/usr/include/c++/9/ext/atomicity.h:96
#6  0x00000000005f2e6f in non-virtual thunk to QtConcurrent::RunFunctionTask<bool>::run() () at /usr/local/include/QtCore/qarraydata.h:220
#7  0x00007ffff3602ac2 in ?? () from /tmp/.mount_Olive-z2HyjO/usr/bin/../lib/libQt5Core.so.5
#8  0x00007ffff35ff7d3 in ?? () from /tmp/.mount_Olive-z2HyjO/usr/bin/../lib/libQt5Core.so.5
#9  0x00007ffff4ba0590 in start_thread (arg=0x7fffd1ffb640) at pthread_create.c:463
#10 0x00007ffff3125223 in clone () at ../sysdeps/unix/sysv/linux/x86_64/clone.S:95
```
</code></pre></details>

**Comment by Simran-B** [comment]
Does it occur if you render other footage and sequences as well or just with a particular one?

**Comment by RachitKeertiDas** [comment]
It does not occur for all sequences, but it occurs regularly though.
It's more prevalent when the video track is not completely filled.

**Comment by itsmattkc** [comment]
Does this still occur in the latest commit?

**Comment by itsmattkc** [comment]
Closing due to inactivity under the assumption it has been resolved along with #1418.
